setup.py

```diff
@@ -10,6 +10,7 @@
 ENTRY_POINTS = {
     'pipeline.handlers': [
         'AatamsNrtHandler = aodndata.aatams.aatams_nrt:AatamsNrtHandler',
+        'AcornHandler = aodndata.acordn.handler:AcornHandler',
```

Comment:
We need 2 distinct pipelines. One for RT and one for DM. The reason is DM files can be uploaded per thousands and take time to be processed, so in the meantime we don't want the RT flow to be affected.

Reply:
Just chiming in on this one... this package only provides the *handler*, and so this package only needs to worry about that. The split directories for RT and DM and the different regexes should be reflected in the *pipeline* configuration rather than the handler definition here.

We'll need to look at how to most cleanly map the existing RT/DM over, which won't be too hard.